sqlglot/generator.py

```diff
@@ -233,6 +233,9 @@ class Generator:
     # Whether or not nested CTEs (e.g. defined inside of subqueries) are allowed
     SUPPORTS_NESTED_CTES = True
 
+    # Whether or not conditions require booleans WHERE x = 0 vs WHERE x
+    ENSURE_BOOLS = False
```

Comment:
Is this also "0/1 can be used as true/false"?

Reply:
sorry what does this mean?

Reply:
nvm